VCProject.Base/VCHelper.cs

```diff
@@ -41,6 +41,12 @@ private VCFileConfiguration GetVCFileConfig(Document document, out string reason
                 return null;
             }
 
+            if (vcFile.FileType == eFileType.eFileTypeCppHeader)
+            {
+                reasonForFailure = ""; // Empty string == skip output for it.
+                return null;
+            }
+
             if (vcFile.FileType != eFileType.eFileTypeCppCode && vcFile.FileType == eFileType.eFileTypeCppClass)
```

Comment:
I think this line is actually wrong, can't make any sense of the "&& vcFile.FileType == eFileType.eFileTypeCppClass".
Can we instead just have a single condition like this:
 if (vcFile.FileType != eFileType.eFileTypeCppCode)
I think this should catch all headers as well. I'd like to keep a meaningful value for reasonForFailure. If you really want to suppress the message we could also just introduce an enum with the various failure states that can be resolved to a string (to filter for "uninteresting" errors)